TSOClient/tso.content/ContentSrc/Fonts/simdialogue.ini

```diff
@@ -1,6 +1,6 @@
-﻿[font]
-path=Fonts/kerned-tso.ttf
+[font]
+path=Fonts/SimDialogueReBold.ttf
 [characters]
 ;See https://unicode-table.com/en for a nice overview of characters available in unicode
 ;Include space, and all standard printable ASCII characters and unicode arrows
-ranges=( ,~),(¡,£),(¥,¥),(§,¬),(®,±),(´,¸),(º,»),(¿,Ā),(Œ,œ),(Š,š),(Ÿ,Ÿ),(Ž,ž),(ƒ,ƒ),(ˆ,ˆ),(˜,˜),(Α,Ρ),(Σ,Ω),(α,ρ),(σ,ω),(Ё,Ё),(А,я),(ё,ё),(‐,‐),(–,―),(‘,‚),(“,„),(†,•),(‥,…),(‰,‰),(′,″),(‹,※),(€,€),(℃,℃),(№,№),(℡,™),(Å,Å),(Ⅰ,Ⅹ),(ⅰ,ⅹ),(←,↓),(⇒,⇒),(⇔,⇔),(∀,∀),(∂,∃),(∇,∈),(∋,∋),(∑,∑),(√,√),(∝,∠),(∥,∥),(∧,∬),(∮,∮),(∴,∵),(∽,∽),(≒,≒),(≠,≡),(≦,≧),(≪,≫),(⊂,⊃),(⊆,⊇),(⊥,⊥),(⊿,⊿),(⌒,⌒),(①,⑳),(─,┃),(┌,┌),(┏,┏),(┐,┐),(┓,└),(┗,┘),(┛,┝),(┠,┠),(┣,┥),(┨,┨),(┫,┬),(┯,┰),(┳,┴),(┷,┸),(┻,┼),(┿,┿),(╂,╂),(╋,╋),(■,□),(▲,△),(▼,▽),(◆,◇),(○,○),(◎,●),(◯,◯),(★,☆),(♀,♀),(♂,♂),(♪,♪),(♭,♭),(♯,♯),(　,〕),(〝,〟)
\ No newline at end of file
+ranges=( ,~),( ,±),(´,¸),(º,»),(¿,ň),(Ŋ,ſ),(Ə,Ə),(Ơ,ơ),(Ư,ư)(Ș,ț),(Ȳ,ȳ),(ȷ,ȷ),(ə,ə),(ˆ,ˇ),(˘,˝),(̉,̉),(̒,̓),(̛,̛),(̣,̣),(̦,̦),(;,;),(΄,Ί),(Ό,Ό),(Ύ,Ρ),(Σ,ώ),(Ѐ,џ),(Ґ,ғ),(Җ,ҝ),(Ҡ,ң),(Ҫ,ҫ),(Ү,ҳ),(Ҷ,һ),(Ӏ,ӂ),(ӏ,ӑ),(Ӗ,ә),(Ӣ,ӣ),(Ө,ө),(Ӯ,ӯ),(Ӳ,ӳ),(Ạ,ỹ),(‐,―),(‘,‣),(‥,…),(‰,‰),(′,″),(‹,※),(€,€),(⃘,⃘),(℃,℃),(№,№),(℡,™),(Å,Å),(Ⅰ,ⅿ),(←,↕),(⇒,⇒),(⇔,⇔),(∀,∀),(∂,∃),(∇,∈),(∋,∋),(∑,∑),(√,√),(⇒,⇒),(⇔,⇔),(∀,∀),(∂,∃),(∇,∈),(∋,∋),(∑,∑),(√,√),(∝,∠),(∥,∥),(∧,∬),(∮,∮),(∴,∵),(∽,∽),(≈,≈),(≒,≒),(≠,≡),(≤,≧),(≪,≫),(⊂,⊃),(⊆,⊇),(⊥,⊥),(⊿,⊿),(⌒,⌒),(①,⑳),(─,┃),(┌,┌),(┏,┏),(┐,┐),(┓,└),(┗,┘),(┛,┝),(┠,┠),(┣,┥),(┨,┨),(┫,┬),(┯,┰),(┳,┴),(┷,┸),(┻,┼),(┿,┿),(╂,╂),(╋,╋),(■,□),(▲,△),(▼,▽),(◆,◇),(◊,○),(◎,●),(◯,◯),(★,☆),(♀,♀),(♂,♂),(♪,♪),(♭,♭),(♯,♯),(　,〕),(〝,〟),(,)
```

Comment:
Missing comma here: `(Ư,ư)(Ș,ț)`

The MSDFExtension also has trouble importing this file, but that was on my end and will be fixed with a commit to master. For some reason it was forcing read as ASCII rather than UTF-8 (how did thai characters work?)

Reply:
Added the missing comma and removed a private range that I guess causes problems. But I don't know how the Thai characters will work now since they contain references to characters in the private range.